### PR TITLE
Moulin cr7

### PR DESCRIPTION
**Pull request by iusyk**

### CODE REVIEW

File: meta-xt-driver-domain/recipes-core/images/core-image-minimal.bbappend [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+IMAGE_INSTALL_append = " \
```
Comment (lorc) [inline meta-xt-driver-domain/recipes-core/images/core-image-minimal.bbappend:2]: I can see that this is a copy of core-image-weston.bbappend.

I believe it is better to create .inc file with this contents and include it from both .bbappends

File: meta-xt-domx/conf/machine/h3ulcb-4x2g-ab-xt.conf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
```
Comment (lorc) [inline meta-xt-domx/conf/machine/h3ulcb-4x2g-ab-xt.conf:1]: Are you actually need to describe a machine? Prod devel works without this

Comment (iusyk) [reply meta-xt-domx/conf/machine/h3ulcb-4x2g-ab-xt.conf:1]: > 

really, if to leave the support just a machine (what is true), than, this configuration can be removed.
i'wll change it

**Comment by iusyk** [comment]
not relevant

**Pull request closed without merging**